include/oneapi/dpl/pstl/numeric_impl.h

```diff
@@ -217,7 +212,8 @@ __brick_transform_scan(_RandomAccessIterator __first, _RandomAccessIterator __la
 
 template <class _RandomAccessIterator, class _OutputIterator, class _UnaryOperation, class _Tp, class _BinaryOperation,
           class _Inclusive>
-typename ::std::enable_if<is_arithmetic_udop<_Tp, _BinaryOperation>::value, ::std::pair<_OutputIterator, _Tp>>::type
+typename ::std::enable_if<!oneapi::dpl::__unseq_backend::is_arithmetic_plus<_Tp, _BinaryOperation>::value,
```

Comment:
Actually, we have such check in `__simd_scan` bricks.
For arithmetic and `std::plus` we have openMP SIMD API and have got the fast vectorized code, otherwise - also vectorized but with specific requirements for default constructor for type, due to "User Defined Scan" in  openMP SIMD API has such requirement. A am not sure that we should avoid vectorized brick call here.